rust_src/src/fastjsonrpc.rs

```diff
@@ -0,0 +1,125 @@
+extern crate remacs_generated;
```

Comment:
What is the purpose of this jsonrpc code, and why is it included in this merge request?

Reply:
So, this PR is just POC to prove that having bindings in a separate crate will speedup language server support(and `cargo check` in general) and that file is just WIP of one of the features that is going to be implemented. It is also a reflection of offline discussions between me and @brotzeit . 

ATM we think that the following crate structure will be most appropriate (it is not reflected in this PR)

- generated stuff
- lisp library e. g. the functions needed for rust-emacs interoperability
- main crate (which packages everything)
- feature 1..N 

The purpose of this is to:
1. Avoid the need to work in the generated stuff crate due to poor IDE setup.
2. Ability to make feature 1..N optional easily.

Let me know if this makes sense to you.

Reply:
That's an excellent summary; it should be in at least one of the commit messages! However, it doesn't quite explain what the jsonrpc code is for, and why it's included in the same merge request as the changes for splitting the code into a new crate.

Reply:
Converted it to draft. This is not something that will stay as it is(ftr nothing in this repo will stay as it is) Once lisp_fn effort is finished we will rebase and most likely squash the commits in one commit so we can have a clean base to work on.